### PR TITLE
community/events: Update SoCal schedule and Cephalocon Sponsors

### PR DESCRIPTION
**Pull request by Thingee**

### CODE REVIEW

**Review by anthonyeleven — APPROVED** [review]

**Comment by anthonyeleven** [comment]
Check failed, with no identified problem.  🤷‍♀️

File: src/en/community/events/2023/cephalocon-amsterdam/index.md [hunk 1]
```diff
@@ -8,6 +8,11 @@ image: "/assets/bitmaps/events/2023/cephalocon/banner.png"
 tags:
   - cephalocon
 sponsors:
+  - Label: Silver
```
Comment (dvanders) [inline src/en/community/events/2023/cephalocon-amsterdam/index.md:11]: "Label" -> "label" ?

Comment (dvanders) [reply src/en/community/events/2023/cephalocon-amsterdam/index.md:11]: @Thingee ^^ is this the reason it didn't compile?

File: src/en/community/events/2023/ceph-days-socal/index.md [hunk 2]
```diff
@@ -156,22 +147,13 @@ IBM</center>
   <tr>
    <td>1:00 PM
    </td>
-   <td><strong>Integrating third party software in Ceph Testing</strong>
-<p>
-In this talk we discuss how third party software modules are integrated with ceph components. The main aim to do this is to test every component as a group. This helps us find problems when different software modules interact with each other. We’d discuss how this testing is done in Ceph using its test automation framework (i.e) Teuthology. This includes how a task is written in order to get it tested in the environment.
+   <td><strong<TBD</strong>
```
Comment (dvanders) [inline src/en/community/events/2023/ceph-days-socal/index.md:150]: Found it "<strong" -> "<strong>"

```
`TemplateWriterWriteError` was thrown
> Parse Error: <strong<TBD</strong>
```

**Pull request merged**